src/panels/NightSkyPanel/tabs/NightSkySolarSystemTab.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button, Group, Text } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { BoolInput } from '@/components/Input/BoolInput';
+import { useProperty } from '@/hooks/properties';
+import { MoonPerformShadingKey } from '@/util/keys';
+
+export function NightSkySolarSystemTab() {
+  const luaApi = useOpenSpaceApi();
+
+  const [performShading, setPerformShading] = useProperty(
+    'BoolProperty',
+    MoonPerformShadingKey
+  );
+  const [moonScale, setMoonScale] = useProperty(
+    'DoubleProperty',
+    'Scene.Moon.Scale.Scale'
+  );
+
+  function action(identifier: string): void {
+    luaApi?.action.triggerAction(identifier);
+  }
+
+  return (
+    <>
+      <Text size={'xl'} my={'md'}>
+        Trails
+      </Text>
+      <Group mb={'md'}>
+        <Button
+          onClick={() => {
+            action('os.FadeDownTrails');
+          }}
+        >
+          Hide ALL Trails
+        </Button>
+        <Button
+          onClick={() => {
+            luaApi?.fadeIn('{planetTrail_solarSystem}');
+            luaApi?.fadeOut('Scene.EarthTrail.Renderable');
+          }}
+        >
+          Show Planet Trails
+        </Button>
+      </Group>
+
+      <Text size={'xl'} my={'md'}>
+        Labels
+      </Text>
+      <Group>
+        <Button onClick={() => luaApi?.fadeIn('{solarsystem_labels}.Renderable')}>
+          Show Labels
+        </Button>
+        <Button onClick={() => luaApi?.fadeOut('{solarsystem_labels}.Renderable')}>
+          Hide Labels
+        </Button>
+      </Group>
+
+      <Text size={'xl'} my={'md'}>
+        Planets
+      </Text>
+      <Group>
+        <Button onClick={() => action('os.nightsky.ShowNightSkyPlanets')}>
+          Show Night Sky Planets
+        </Button>
+        <Button onClick={() => action('os.nightsky.HideNightSkyPlanets')}>
+          Hide Night Sky Planets
+        </Button>
+      </Group>
+
+      <Text size={'xl'} my={'md'}>
+        Moon
+      </Text>
+      <Group>
+        <Button onClick={() => setMoonScale(1.0)}>Default Scale Moon (1x)</Button>
+        <Button onClick={() => setMoonScale(2.0)}>Enlarge Moon (2x)</Button>
+        <Button onClick={() => setMoonScale(4.0)}>Enlarge Moon (4x)</Button>
+        <Button
+          onClick={() => {
+            if (moonScale) {
```

Comment:
this check is problematic, since it will also be false if moonScale is zero and then the value will not change. Same issue below

Instead, check specifically for `undefined`
```suggestion
            if (moonScale !== undefined) {
```
or maybe even better, add a guard that only shows these settings if this setting is available: 
```
        {moonScale !== undefined ? (
          <>
            <Button onClick={() => setMoonScale(1.0)}>Default Scale Moon (1x)</Button>
            <Button onClick={() => setMoonScale(2.0)}>Enlarge Moon (2x)</Button>
            <Button onClick={() => setMoonScale(4.0)}>Enlarge Moon (4x)</Button>
            <Button onClick={() => setMoonScale(moonScale + 0.5)}>+</Button>
            <Button onClick={() => setMoonScale(moonScale - 0.5)}>-</Button>
          </>
        ) : (
          <Text>Could not find Moon scale settings</Text>
        )}
```

This would also connect to the issue that currently this panel needs to account for if certain content isn't loaded (in which case it should still work and not seem "broken")